src/cli/cli.cc

```diff
@@ -831,6 +832,18 @@ int main (const int argc, const char* argv[]) {
         }
       }
 
+      auto host = optionValue(arg, "--host");
+      if (host.size() > 0) {
+        devHost = host;
+      } else {
+        if (flagBuildForIOS || flagBuildForAndroid) {
+          auto r = exec("ifconfig | grep -w 'inet' | awk '!match($2, \"^127.\") {print $2; exit}'");
+          if (r.exitCode == 0) {
+            devHost = r.output;
+          }
```

Comment:
For iOS and Android builds, I try to infer the LAN IP of the dev machine.

Reply:
Not sure if that command works on anything but macOS though

Reply:
it works on macOS without the escaping of the double quotes. This worked for me

```bash
ifconfig | grep -w 'inet' | awk '!match($2, "^127.") {print $2; exit}'
```

Reply:
Agreed. The escaping is for the `exec` call :)